packages/salesforcedx-vscode-lwc/resources/dark/debug.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16.7 16.7"><defs><style>.cls-1{fill:none;stroke:#c5c5c5;stroke-miterlimit:10;stroke-width:0.7px;}</style></defs><title>Asset 1</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M4.63,4.75A4.66,4.66,0,0,1,6.33,1,3.68,3.68,0,0,1,9,.41,3.75,3.75,0,0,1,10.32,1a4.66,4.66,0,0,1,1.75,3.74Z"/><path class="cls-1" d="M9.28,4.85c1.3,0,2.6,0,3.89,0a.76.76,0,0,0,.68-.32c.23-.28.49-.55.74-.82a.44.44,0,0,1,.73,0,.54.54,0,0,1,0,.8c-.35.38-.7.77-1.07,1.13a.66.66,0,0,1-.42.17c-.35,0-.69,0-1,0L14.24,9h1.45c.42,0,.66.21.66.57s-.23.57-.66.57H14.31l-.59,2.66c.55-.09.81.46,1.17.82s.43.47.64.72a.54.54,0,0,1,0,.77.44.44,0,0,1-.73,0c-.25-.26-.5-.54-.74-.81a.91.91,0,0,0-1.41,0,6.37,6.37,0,0,1-3.38,2l-.4.05H8l-.41-.05a6.41,6.41,0,0,1-3.43-1.93.91.91,0,0,0-1.41,0L2,15.22a.45.45,0,0,1-.74,0,.55.55,0,0,1,0-.77c.21-.25.42-.5.64-.73s.61-.91,1.16-.84c-.2-.88-.4-1.73-.61-2.64H1c-.44,0-.67-.2-.68-.56s.23-.58.66-.58H2.47L3.9,5.84c-.31,0-.66,0-1,0a.76.76,0,0,1-.44-.16c-.37-.36-.73-.75-1.08-1.12a.54.54,0,0,1-.06-.8.45.45,0,0,1,.74,0c.26.27.52.53.75.81a.76.76,0,0,0,.69.31l3.93,0H9.28Z"/></g></g></svg>
```

Comment:
Where did we get these images from ?

Reply:
Maggie Li (UX Designer from Sales Cloud) created the icons for debug/startWatching/stopWatching. I copied the other icons from the [apex extension](https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/packages/salesforcedx-vscode-apex/resources/light) since I wanted to keep the test explorer UI consistent.